client/components/MainImage.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import styled from 'styled-components';
+
+class MainImage extends React.Component {
+  //for future adjustments, I am making this a class
+  constructor (props) {
+    super(props);
+    this.state = {
+    }
+  }
+
+  render() {
+    let product_data = this.props.product_data;
+    let mainImages = [];
+
+    product_data.map((product, i) => {
+      let productImg = product.image_loc;
+      mainImages.push(
+        <MainImageSlide id={`main_image_${i}`}>
+          <Image
+            key={product.product_name + product.ID}
+            src={productImg}
+            onClick={(e) => this.props.overlayHandleClick(e)}
+          ></Image>
+        </MainImageSlide>
+      )
+    })
+
+    return (
+      <MainImageContainer
+        key={product_data.product_name + 'main'}
```

Comment:
It looks like you were trying to clean up and make all the props value line up in one single. Maybe do it for this item as well?